.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build install ISPC action
+
```

Comment:
Too many vertical indentation (two empty lines) to my taste, but no strong opinion here.

Reply:
Fixed